exporter/prometheusexporter/end_to_end_test.go

```diff
@@ -116,11 +117,9 @@ func TestEndToEndSummarySupport(t *testing.T) {
 	}
 	t.Cleanup(func() { require.NoError(t, prometheusReceiver.Shutdown(ctx)) })
 
-	// 4. Scrape from the Prometheus exporter to ensure that we export summary metrics
-	// We shall let the Prometheus exporter scrape the DropWizard mock server, at least 9 times.
-	for i := 0; i < 8; i++ {
-		<-waitForScrape
-	}
+	// 4. Scrape from the Prometheus receiver to ensure that we export summary metrics
```

Comment:
We are scraping from the prometheus exporter, not receiver.

Reply:
The scrapes come from the receiver.  That is what the `wg.Wait()` is waiting for.  Once the receiver has scraped the target enough times then the wait group is resolved.